### ISSUE TITLE
too many open files

### ISSUE THREAD

**Issue by gtr32x** [influxdata/influxdb#21003]
I've currently running InfluxDB 2.0.4 on Mac Big Sur. I'm loading a lot of historical finance data into Influx with a multi year span. However after the first year of loading I am immediately hit with a `too many open files` issue.

After looking online for solutions, I've came across the solution of setting `ulimit` or setting shard duration during bucket creation.

When I tried to change `ulimit` in mac, I can see with `ulimit -n` getting 65536, or `launchctl limit maxfiles` showing 65536, this did not avert the problem.

When I tried to set shard duration in InfluxDB v2, I realized that no docs specify this possibility in v2 and it appears to be a v1 possibility.

Basically now I am wondering what have I done wrong or how can I amend this issue?

Thank you.

**Comment by danxmoran** [comment]
@gtr32x 2.0.5 (to be released by end of this month) will restore the ability to set shard-group durations

**Comment by gtr32x** [comment]
Thank you for the reply, is there a way I can get the nightly a bit earlier?

**Comment by danxmoran** [comment]
@gtr32x you can pull the nightly archive from our [Downloads Portal](https://portal.influxdata.com/downloads/) (see the "2.x Nightly" button)

**Comment by gtr32x** [comment]
Thank you
